### PR TITLE
WIP:DOTNET zstream onboarding on ibmcloud

### PR DESCRIPTION
**Pull request by smatula**

### CODE REVIEW

File: ci-operator/config/sclorg/ansible-tests/sclorg-ansible-tests-master__dotnet-ocp4.14-lp-interop.yaml [hunk 1]
```diff
@@ -57,6 +57,35 @@ tests:
     test:
     - ref: dotnet-deploy-and-test
     workflow: firewatch-ipi-aws
+- as: dotnet-interop-ibmcloud
+  cron: 0 6 25 12 *
+  steps:
+    cluster_profile: ibmcloud-cspi-qe
+    env:
+      BASE_DOMAIN: zstream.interop.ccitredhat.com
+      DOTNET_VERSION: dotnet_60
```
Comment (Simranpal) [inline ci-operator/config/sclorg/ansible-tests/sclorg-ansible-tests-master__dotnet-ocp4.14-lp-interop.yaml:66]: Can this be moved out of config to dockerfile or somewhere else

Comment (smatula) [reply ci-operator/config/sclorg/ansible-tests/sclorg-ansible-tests-master__dotnet-ocp4.14-lp-interop.yaml:66]: We talking about the DOTNET_VERSION... Required env by the step.

**Review by Simranpal — APPROVED** [review]
/lgtm

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-sclorg-ansible-tests-master-dotnet-ocp4.15-lp-interop-dotnet-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-sclorg-ansible-tests-master-dotnet-ocp4.15-lp-interop-dotnet-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-sclorg-ansible-tests-master-dotnet-ocp4.14-lp-interop-dotnet-interop-ibmcloud

**Pull request closed without merging**